scripts/globals/spells/utsusemi_ichi.lua

```diff
@@ -11,8 +11,9 @@ end
 
 function onSpellCast(caster, target, spell)
     local effect = target:getStatusEffect(tpz.effect.COPY_IMAGE)
-
-    -- Get extras shadows
+    and target:delStatusEffect(67) -- Third Eye and Utsusemi don't stack.  Utsusemi removes Third Eye.
```

Comment:
This section will resolve to `local effect = target:getStatusEffect(tpz.effect.COPY_IMAGE) and target:delStatusEffect(67)`, making effect a bool (I think). Is that your intention?

Or should the `and` be removed?
```lua
local effect = target:getStatusEffect(tpz.effect.COPY_IMAGE)
target:delStatusEffect(67) -- Third Eye and Utsusemi don't stack.  Utsusemi removes Third Eye.
```